packaging_automation/update_docker.py

```diff
@@ -59,17 +61,17 @@ def update_docker_file_alpine(project_version: str, template_path: str, exec_pat
     write_to_file(content, dest_file_name)
 
 
-def update_docker_file_for_postgres12(project_version: str, template_path: str, exec_path: str):
+def update_docker_file_for_postgres12(project_version: str, template_path: str, exec_path: str, postgres_version: str):
     content = process_template_file(project_version, template_path,
-                                    docker_templates[SupportedDockerImages.postgres12])
+                                    docker_templates[SupportedDockerImages.postgres12], postgres_version)
     dest_file_name = f"{exec_path}/{docker_outputs[SupportedDockerImages.postgres12]}"
     write_to_file(content, dest_file_name)
 
 
-def update_docker_file_for_postgres14(project_version: str, template_path: str, exec_path: str):
+def update_docker_file_for_postgres13(project_version: str, template_path: str, exec_path: str, postgres_version: str):
     content = process_template_file(project_version, template_path,
-                                    docker_templates[SupportedDockerImages.postgres14])
-    dest_file_name = f"{exec_path}/{docker_outputs[SupportedDockerImages.postgres14]}"
+                                    docker_templates[SupportedDockerImages.postgres13], postgres_version)
+    dest_file_name = f"{exec_path}/{docker_outputs[SupportedDockerImages.postgres13]}"
     dir_name = os.path.dirname(dest_file_name)
     if not os.path.exists(dir_name):
         os.makedirs(dir_name)
```

Comment:
Why do we create missing directories only for pg13 and not pg12?

Can you move the common logic in a function, and reduce repetition here?

Reply:
Fixed as you suggested